osu.Game/Beatmaps/BeatmapModelManager.cs

```diff
@@ -85,6 +90,24 @@ public virtual void Save(BeatmapInfo beatmapInfo, IBeatmap beatmapContent, ISkin
             WorkingBeatmapCache?.Invalidate(beatmapInfo);
         }
 
+        /// <summary>
+        /// Add a new difficulty to the beatmap set represented by the provided <see cref="BeatmapSetInfo"/>.
+        /// </summary>
+        public BeatmapInfo AddDifficultyToBeatmapSet(BeatmapSetInfo beatmapSetInfo, Beatmap beatmap)
+        {
+            return Realm.Run(realm =>
```

Comment:
This `Realm.Run` usage doesn't seem to be required, nor is it adding any kind of transactional safety. Oversight maybe?

Reply:
Looks like a vestige that I forgot to rebase out. I'll double check for sure though.